pkg/controller/crdmirroring/crdmirroring_handlers.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crdmirroring
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/controller/crdmirroring/types"
+)
+
+func (c *Controller) onNewCRDAdd(obj interface{}) {
+	CRD := obj.(metav1.Object)
+
+	if CRD == nil {
```

Comment:
Why is this added? I think it's impossible to get this and never see this happen

Reply:
I got this from endpointslice mirroring handler function, so I added it.